src/main/java/stsjorbsmod/cards/wanderer/Amnesia.java

```diff
@@ -38,9 +41,19 @@ public Amnesia() {
 
     @Override
     public void use(AbstractPlayer p, AbstractMonster _) {
-        if(dontTriggerOnUseCard) {
-            addToBot(new SnapAction(p));
-            addToBot(new RemovePowersMatchingPredicateAction(p, power -> power.type == AbstractPower.PowerType.BUFF || power.type == AbstractPower.PowerType.DEBUFF));
+        if (dontTriggerOnUseCard) {
+            List<AbstractPower> buffs = p.powers.stream()
+                    .filter(power -> power.type == AbstractPower.PowerType.BUFF)
+                    .collect(Collectors.toList());
+            if (buffs.size() == 0) {
+                AbstractDungeon.actionManager.addToBottom(new TalkAction(true,"~...~", 2.0f, 2.0f));
+            } else if (p.hasPower(ArtifactPower.POWER_ID)) {
+                AbstractDungeon.actionManager.addToBottom(new ReducePowerAction(p, p, ArtifactPower.POWER_ID, 1));
```

Comment:
It might be more consistent looking for this to include the other effects that happen when Artifact triggers during power application (see `ApplyPowerAction`), something like:

```suggestion
                    addToBot(new TextAboveCreatureAction(this.target, EXTENDED_DESCRIPTION[0]));
                    CardCrawlGame.sound.play("NULLIFY_SFX");
                    p.getPower(ArtifactPower.POWER_ID).flashWithoutSound();
                    p.getPower(ArtifactPower.POWER_ID).onSpecificTrigger();
```

Reply:
Makes sense. I'll add this.